app/src/main/java/org/simple/clinic/home/overdue/OverdueAppointment.kt

```diff
@@ -77,8 +77,8 @@ import java.util.UUID
             P.deletedAt IS NULL
             AND P.status != 'dead'
             AND A.deletedAt IS NULL
+            AND PPN.deletedAt IS NULL
```

Comment:
Are we sure we want to ignore patients whose phone number is deleted? The phone number might be deleted, but the patient might be still active. Can we verify this case once, please?

Reply:
Yeah. The phone number can not be deleted once added unless all the app data is cleared.